src/uct/ib/rdmacm/rdmacm_cm_ep.h

```diff
@@ -41,10 +42,14 @@ typedef struct uct_rdmacm_cm_ep {
 enum {
     UCT_RDMACM_CM_EP_ON_CLIENT     = UCS_BIT(0),
     UCT_RDMACM_CM_EP_ON_SERVER     = UCS_BIT(1),
-    UCT_RDMACM_CM_EP_CONNECTED     = UCS_BIT(2),
-    UCT_RDMACM_CM_EP_DISCONNECTING = UCS_BIT(3) /* uct_ep_disconnect was called
-                                                   on the ep. this ep is not
-                                                   necessarily disconnected yet */
+    UCT_RDMACM_CM_EP_CONNECTED     = UCS_BIT(2), /* connection was successfully
+                                                    established */
+    UCT_RDMACM_CM_EP_DISCONNECTED  = UCS_BIT(3), /* got disconnect event */
+    UCT_RDMACM_CM_EP_DISCONNECTING = UCS_BIT(4), /* uct_ep_disconnect was called
```

Comment:
these two flags have very similar names...

Reply:
could you pls propose better?